addon/mixins/dirty-confirm-route.js

```diff
@@ -34,6 +34,9 @@ export default Ember.Mixin.create({
         case ROLLBACK: return true;
         case BUBBLE: return true;
       }
+    },
+    rollbackModel: function(model) {
+      this.checkModelForDirty(model, null, null, true);
```

Comment:
Hitting cancel on a form is also an action taken by the user. Why make the cases different?